### ISSUE TITLE
Install error using "Getting Started" directions

### ISSUE THREAD

**Issue by neely** [Noble-Lab/casanovo#147]
Being the naive user, I followed the [directions ](https://casanovo.readthedocs.io/en/latest/getting_started.html) using my anaconda3 prompt (no pytorch install since it said optional), and when I tested the install with a help command I got the following.

```
(casanovo_env) C:\Users\ban>casanovo --help
Traceback (most recent call last):
  File "C:\Users\ban\Anaconda3\envs\casanovo_env\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\ban\Anaconda3\envs\casanovo_env\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\ban\Anaconda3\envs\casanovo_env\Scripts\casanovo.exe\__main__.py", line 4, in <module>
  File "C:\Users\ban\Anaconda3\envs\casanovo_env\lib\site-packages\casanovo\casanovo.py", line 21, in <module>
    from pytorch_lightning.lite import LightningLite
ModuleNotFoundError: No module named 'pytorch_lightning.lite'
```

I assume this is something simple, and likely my fault, but figured I could be the naive tester for these instructions.

**Comment by asalt** [comment]
This broke with the new release of pytorch lightning 2.0. At the moment, you can solve the issue by installing pytorch lightning 1.7

`pip install pytorch_lightning==1.7`

**Comment by bittremieux** [comment]
There are indeed some breaking changes with the latest PyTorch Lightning release. We'll have to upgrade some small pieces in our code or pin the dependency to a lower version. This will be fixed in the next release.

**Comment by neely** [comment]
Thanks! Issue is solved by installing pytorch lightning 1.7, which also I might say shouldn't be an optional step in the [getting started](https://casanovo.readthedocs.io/en/latest/getting_started.html) directions. But is working for me now.

**Comment by bittremieux** [comment]
PyTorch being optional refers to installing it manually to ensure compatibility with your GPU. I agree that it's a bit confusing as written right now, so we should rephrase it a bit. I'll reopen this issue for the moment so we can properly track this.
